README.md

```diff
@@ -15,7 +15,7 @@ npm i @autotelic/fastify-opentelemetry
 // to fastify-opentelemetry. (See the example configuration below.)
 require('./openTelemetryConfig')
 const openTelemetryPlugin = require('@autotelic/fastify-opentelemetry')
-const fastify = require('fastify')()
+const fastify = require('fastify')();
```

Comment:
Is the `;` meant to be here in this code example?

Reply:
Because the next line of code is an IIFE, we need the semi-colon to prevent it from being treated as a function argument.